manager/manager.go

```diff
@@ -100,6 +101,51 @@ func (m *Manager) ListInstances(limit int64, tagFilters []*TagFilter) ([]*Instan
 	return out, nil
 }
 
+// ListDocuments fetches a list of documents managed by SSM. Paginates until all responses have been collected.
+func (m *Manager) ListDocuments(limit int64, documentFilters []*ssm.DocumentFilter) ([]*DocumentIdentifier, error) {
+	var out []*DocumentIdentifier
+
+	input := &ssm.ListDocumentsInput{
+		MaxResults:         &limit,
+		DocumentFilterList: documentFilters,
+	}
+
+	for {
+		response, err := m.ssmClient.ListDocuments(input)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to list document")
+		}
+
+		// NOTE: ec2Info will be a shorter list when filtering is applied.
+		for k := range response.DocumentIdentifiers {
+			out = append(out, NewDocumentIdentifier(response.DocumentIdentifiers[k]))
```

Comment:
With `for _, document := range ...` you can just write `NewDocumentIdentifier(document)` (don't need `[k]` even if it's a map)

Reply:
Yepp, fixing.